docs/src/features.md

```diff
@@ -7,26 +7,80 @@
+## [Flexible Connection of Energy Assets](@id flex-asset-connection)
+
+The representation of the energy system in *TulipaEnergyModel.jl* is based on [Graph Theory](https://www.britannica.com/topic/graph-theory), which deals with connection amongst vertices by edges. This representation provides a more flexible framework to model *energy assets* in the system as *vertices* and *flows* between energy assets as *edges*. In addition, it poses some advantages from the modeling perspective. For instance, connecting assets directly to each other without having a node between them allows for a reduction of the number of variables and constraints to represent different configurations (e.g., co-location of energy storage and renewable units to model a hybrid operation).
+
+Consider the following example to demonstrate the benefits of this approach. In the classic connection approach, the nodes play a crucial role in modeling. For example, every asset needs to be connected to a node that has balance constraints attached to it. Therefore, to include the co-location of a storage asset and a renewable asset, we must introduce a node between these assets and the external power grid, as shown in the following figure:
+
+![Classic connection](./figs/flexible-connection-1.png)
+
+In this system, the `phs` storage asset charges and discharges from the connection `node`, while the `wind` turbine produces power that also goes directly to the `node`. This `node` is connected to the external power grid through a transmission line that leads to the `balance` node. Essentially, the `node` acts as a balancing point for the assets in this hybrid operation. Furthermore, these hybrid operations impose an additional constraint to ensure that storage charges from the power grid are avoided. The section with [`comparison of different modeling approaches`](@ref comparison) shows the quantification of these reductions.
+
+Let's consider the modeling approach in *TulipaEnergyModel.jl*. As nodes are no longer needed to connect assets, we can connect them directly to each other as shown in the figure below:
+
+![Flexible connection](./figs/flexible-connection-2.png)
+
+By implementing this approach, we can reduce the number of variables and constraints involved in the process. For example, the balance constraint in the intermediate node is no longer needed, as well as the extra constraint to avoid the storage charging from the power grid.  Additionally, we can eliminate the variable that determines the flow between the intermediate node and the power grid.
```

Comment:
```suggestion
By implementing this approach, we can reduce the number of variables and constraints involved in the process. For example, the balance constraint in the intermediate node is no longer needed, as well as the extra constraint to avoid the storage charging from the power grid.  Additionally, we can eliminate the variable that determines the flow between the intermediate node and the power grid because the flow from `phs` to `balance` can directly link to the external grid.
```